INSTALL.md

```diff
@@ -0,0 +1,171 @@
+
+If you want to have rakudo, nqp, and moar installed into a different
+directory, you may supply --prefix= to Configure.pl. For example, with:
+
+    $ perl Configure.pl --gen-moar --gen-nqp --backends=moar --prefix=$HOME/raku
+
+all binaries will be found under $HOME/raku/bin:
+
+    $ $HOME/raku/bin/rakudo hello.raku
+
+Simply running "rakudo" will drop you into a REPL (read-eval-print-loop)
+that you can use for exploratory programming:
+
+    $ ./install/bin/rakudo
+
+To find out the version and backend of the current binary one can use
+--version command line argument:
+
+    $ ./install/bin/raku --version
+
+See the manual page ("docs/running.pod") for more about command-line
+options.
+
+If you would like readline-like features in REPL, such as command history,
+line editing, and tab completion for builtins, you should install the
+Linenoise module via recommended Raku package manager zef:
+
+    $ zef install Linenoise
+
+More information about zef itself and how to install it can be found at
+<https://github.com/ugexe/zef>.
+
+The "--gen-moar" above option tells Configure.pl to automatically download
+and build the most appropriate version of NQP and MoarVM into local "nqp/"
+and "nqp/MoarVM/" subdirectories, install NQP and MoarVM into the "install/"
+subdirectory or into the one specified with "--prefix", and use them for
+building Rakudo.  It's okay to use the "--gen-moar" option on later
+invocations of Configure.pl; the configure system will re-build NQP and/or
+MoarVM only if a newer version is needed for whatever version of Rakudo
+you're working with.
+
+The versions of any already installed NQP or MoarVM binaries must satisfy a
+minimum specified by the Rakudo being built -- Configure.pl and "make" will
+verify this for you.  Released versions of Rakudo generally build against
+the latest release of MoarVM; checkouts of Rakudo's HEAD revision from
+GitHub often require a version of MoarVM that is newer than the most recent
+MoarVM monthly release.
+
+# Build/install problems
+
+Occasionally, there may be problems when building/installing Rakudo.  Make
+sure you have a backup of any custom changes you have done to the source
+tree before performing the following steps:
+
+Try to remove the "install/" subdirectory:
+
+    $ cd rakudo
+    $ rm -r install
+    $ git pull
+    $ perl Configure.pl --gen-moar --gen-nqp --backends=moar # for instance
+    $ make install
+
+Or, in case you are really stuck, start with a fresh source tree:
+
+    $ rm -r rakudo
+    $ git clone git://github.com/rakudo/rakudo.git
+
+# Running the test suite
+
+Entering "make test" will run a small test suite that comes bundled with
+Rakudo. This is a simple suite of tests, designed to make sure that the
+Rakudo compiler is basically working and that it's capable of running a
+simple test harness.
+
+Running "make spectest" will import the official Raku test suite from the
+"roast" repository <http://github.com/Raku/roast/> and run all of these
```

Comment:
As long as you're transforming this, could you maybe also markdownify the links?

Reply:
Please, don't. GitHub [supports](https://gist.github.com/VEnis/7465176#automatic-links) the angle brackets for links. For plain text reading they're way more preferable over the original markdown format.